wof/wofpy_config.py

```diff
@@ -14,22 +16,25 @@
 
 Usage:
     wofpy_config INDIR
+    wofpy_config INDIR [--mode=<test,production>]
 
     wofpy_config (-h | --help | -v | --version)
 
 Examples:
-    wofpy_config my_wofpy_config
+    wofpy_config wofpyserver
```

Comment:
@emiliom I am not comfortable with a default directory b/c I think the user must know where s/he is copying the files.
So I added the `wofpyserver` name as a suggestion in the docs, but the user must write that down to make it work.

The script also will fail if that directory already exists. We can add an option o overwrite existing directories later.